config/v1/types_cluster_version.go

```diff
@@ -237,6 +237,13 @@ const (
 	ClusterVersionCapabilityOpenShiftSamples ClusterVersionCapability = "openshift-samples"
 )
 
+const (
```

Comment:
nit, no need to close the previous `const (` block just to open a new one here.  You can drop this new line and the `)` before.